### PR TITLE
report basic summary

### PR DESCRIPTION
**Pull request by rotemtam**
Screen Shot 2022-09-01 at 16 36 14

### CODE REVIEW

**Review by a8m — COMMENTED** [review]
Add screenshots, please.

**Comment by rotemtam** [comment]
Still WIP

File: src/github.ts [hunk 1]
```diff
@@ -69,3 +69,41 @@ export function report(res: AtlasResult): void {
...
+  ]
+  for (const step of steps) {
+    const reports = step.Result?.Reports || []
+    let emoj = '🟢'
+    if (reports.length && !step.Error) {
+      emoj = '🟡'
+    }
+    if (step.Error) {
+      emoj = '🔴'
+    }
```
Comment (a8m) [inline src/github.ts:97]: Can we use the SVG icons instead?

**Review by a8m — APPROVED** [review]
💯

**Pull request merged**